kubernetes/windows/Dockerfile-dev-base-image

```diff
@@ -9,8 +9,7 @@ LABEL vendor=Microsoft\ Corp \
 RUN powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
 # Fluentd depends on cool.io whose fat gem is only available for Ruby < 2.5, so need to specify --platform ruby when install Ruby > 2.5 and install msys2 to get dev tools
```

Comment:
we can remove this Dockerfile-dev-base-image  file since we dont use this.

Reply:
removed